app/core/utils.js

```diff
@@ -2270,7 +2273,15 @@ const getModuleNumberForLevelName = function (courseId, levelName) {
   return moduleNumberByLevelName[levelName] && Number(moduleNumberByLevelName[levelName])
 }
 
+module.exports.aiToolToImage = {
```

Comment:
because actually I don't really see the point of defining const and later add everything to export object again. I think, it is much cleaner this way. It happened to me many times that I defined a new const in utils, and thought it'll be accessible at utils.myVarName, and then I had to come back and add it to exports...